sqlite_query.c

```diff
@@ -369,6 +371,16 @@ sqlite_convert_to_pg(Form_pg_attribute att, sqlite3_stmt * stmt, int stmt_colid,
 				}
 				break;
 			}
+		case VARBITOID:
+		case BITOID:
+			{
+				char * buffer = palloc0(SQLITE_FDW_BIT_DATATYPE_BUF_SIZE);			
+				sqlite3_int64 sqlti = sqlite3_column_int64(stmt, stmt_colid);				
+				buffer = int2binstr(sqlti, buffer, SQLITE_FDW_BIT_DATATYPE_BUF_SIZE);
```

Comment:
Could you separate variable definition part and code part?

Reply:
Fixed in this place and near. I have added empty lines where in code block there is not only `return`. Please refer https://github.com/pgspider/sqlite_fdw/pull/83/commits/ff232874f174af1e29c12d30285bbf3ab512e7c8

Reply:
Please recheck: `sqlite3_int64 sqlti = sqlite3_column_int64(stmt, stmt_colid);`    
it a variable definition.

Reply:
Oh, really. Fixed in https://github.com/pgspider/sqlite_fdw/pull/83/commits/77b0efb62db35bea88b8752d43ad6266bbcfcb00.

Reply:
Thank you. Please help me to resolve this thread.